lib/friendly_shipping/services/ups/parse_shipment_accept_response.rb

```diff
@@ -0,0 +1,161 @@
+# frozen_string_literal: true
+
+require 'dry/monads/result'
+
+module FriendlyShipping
+  module Services
+    class Ups
+      class ParseShipmentAcceptResponse
+        extend Dry::Monads::Result::Mixin
+
+        UPS_SURCHARGE_CODES = {
```

Comment:
At some point it would be helpful to move these large constants into a module, obviating the need to scroll through the entire hash to get to the guts of the class.

Reply:
Yeah, maybe. It's a pain with these long lists of codes and UPS, they're all over the place.

Reply:
Value Objects \o/

Reply:
Or for the time being move those constants to the bottom of the class body. There are not really public constants to read from outside of this class, right? WDYT?